lib/elixir/lib/calendar/datetime.ex

```diff
@@ -109,8 +109,9 @@ defmodule DateTime do
       {:ok, ~U[2016-05-24 13:26:08.003Z]}
 
   When the datetime is ambiguous - for instance during changing from summer
-  to winter time - the two possible valid datetimes are returned. First the one
-  that happens first, then the one that happens after.
+  to winter time - the two possible valid datetimes are returned in a tuple,
+  being its first element the datetime that takes place first, and its second element
```

Comment:
This is a long sentence. We should probably keep the period.

Reply:
Fixed.